parts/k8s/windowskubeletfunc.ps1

```diff
@@ -523,6 +526,8 @@ Update-CNIConfig(`$podCIDR, `$masterSubnetGW)
 
 try
 {
+    `$env:AZURE_ENVIRONMENT_FILEPATH="c:\k\azurestackcloud.json"

```

Comment:
what process uses `AZURE_ENVIRONMENT_FILEPATH` ? I'm wondering why it's set here as an environment variable when I don't see it referenced anywhere

Reply:
The kubelet uses it

Reply:
https://github.com/kubernetes/kubernetes/blob/master/vendor/github.com/Azure/go-autorest/autorest/azure/environments.go#L27

Reply:
Kubernetes has a dependency on Azure's goautorest. In the azure cloud provider code it checks the config file to see which environment it should connect to, and when it realizes it is azure stack cloud it looks for the configuration file stored in the location specified by that env var